paper-icon-button-light.html

```diff
@@ -45,6 +44,12 @@
         opacity: 0.6;
         color: currentColor;
       }
+      
+      img {
+        vertical-align: middle;
+        width: var(--iron-icon-width, 24px);
```

Comment:
I agree that setting this to 24px makes sense but I don't understand the benefit of also providing the variable `--iron-icon-width` here. Wouldn't it make more sense for the user to create a rule that selects and styles the image width directly rather than selecting the paper-icon-button-light and setting `--iron-icon-width`?